.rubocop.yml

```diff
@@ -39,9 +40,13 @@ SketchupRequirements:
 SketchupSuggestions:
   Enabled: false
 
-
+<% if RUBY_VERSION < '2.6' %>
```

Comment:
What syntax is this? YAML? ERB? Is RuboCop performing ERB processing on the config file before evaluating it?